assets/frontend/dashboard/controllers.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('dashboard.portal')
+    .controller('DashBoardController', DashBoardController)
+  ;
+
+  /* DashBoardController
+   * @ desc: This will displat the list of user invoices
+   */
+   function DashBoardController($scope, InvoiceService) {
+    feather.replace();
+    var self = this;
+    var order = 0;
+
+    self.invoiceService = InvoiceService;
+
+    // request to the backend
+    self.sortBy = function(field, order) {
+      self.invoiceService.getList('', field, order);
```

Comment:
wrong. the first parameter in this function is `params` which is a dictionary not a blank string. You put the Query parameter in that param.

DO NOT add unnecessary/duplicate function parameters